Update index.md (#4384)
umbraco/UmbracoDocs: Reference/Configuration/RuntimeSettings/index.md

  "CMS": {
    "Runtime": {
      "MaxQueryStringLength": 90,
      "MaxRequestLength": 2048,
      "Mode": "BackofficeDevelopment"
    }
  }
}
```

`MaxRequsetLength` is specified in kilobytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes, and a maximum query string length of 90 characters.

> `Mode` could have three values: `BackofficeDevelopment` (default), `Development` and `Production`

Reviewer: ```suggestion
`Mode` can have three values: `BackofficeDevelopment` (default), `Development`, and `Production`.
```

> Read more about the details at: [Fundamentals > Setup > Server-Setup > Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes)

Reviewer: ```suggestion
For more information, see the [Runtime modes](/documentation/Fundamentals/Setup/Server-Setup/runtime-modes) article.
```

Author: 👍😁

Reviewer: @idseefeld, 

If you agree with the suggestions, can you click on **Commit Suggestions** -> **Commit changes**. Once done, I'll merge in the PR 😃
